airflow/dag_processing/processor.py

```diff
@@ -522,16 +522,20 @@ def manage_slas(self, dag: DAG, session: Session = None) -> None:
             session.commit()
 
     @staticmethod
-    def update_import_errors(session: Session, dagbag: DagBag) -> None:
+    @internal_api_call("dag_processing.processor.update_import_errors")
```

Comment:
The string `dag_processing.processor.update_import_errors` is redundant with the method name. From `internal_api_call` definition, is not it possible to extract this with something like `func.__module__`?

Reply:
You can, indeed, generate the string `dag_processing.processor.update_import_errors` from the `func` parameter. You can get it with `f"{func.__module__}.{func.__name__}"`. That way, no need to have a parameter to `internal_api_call` function